CONTRIBUTING.md

```diff
@@ -87,7 +87,13 @@ Follow these steps to start contributing:
    $ pip install -e ".[dev]"
    ```
 
-5. Develop the features on your branch.
+5. Install pre-commit hooks, for more details refer to our [pre-commit instructions](./docs/PRECOMMITHOOKS.md):
```

Comment:
I would make this step optional. It should be as easy as possible to get started contributing to lightly and installing pre-commit hooks is an extra step that makes it more complicated. We rely on CI to make sure that everything is working as expected so skipping pre-commit is not a big issue.

Reply:
Would you recommend I just add a "**(Optional)** Install pre-commit hooks, ....." before the step?

Reply:
Yep, sounds good